libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragmentAbstract.java

```diff
@@ -205,6 +205,7 @@ public static MediaType getEditorMimeType(MediaFile mediaFile) {
         void getMention(Consumer<String> onResult);
         void onGutenbergEditorSetStarterPageTemplatesTooltipShown(boolean tooltipShown);
         boolean onGutenbergEditorRequestStarterPageTemplatesTooltipShown();
+        String onFailedToInsertMediaTapped(int mediaId);
```

Comment:
I think I'd rename this to something like `getErrorMessageFromMedia` which would be more explicit.

Reply:
Agree, changed here 9f55e472b2c206adf718755d15498326f58d7c2e